error: use of undeclared identifier 'isatty'

Cannot compile on FreeBSD:

```
[ 83%] Linking CXX shared library libosmpbf.so
[ 83%] Built target osmpbf
[ 91%] Building CXX object tools/CMakeFiles/osmpbf-outline.dir/osmpbf-outline.cpp.o
[ 91%] Built target osmpbf_shared
/tmp/OSM-binary/tools/osmpbf-outline.cpp:97:16: error: use of undeclared identifier 'isatty'
    usecolor = isatty(1);
               ^
1 error generated.
--- tools/CMakeFiles/osmpbf-outline.dir/osmpbf-outline.cpp.o ---
*** [tools/CMakeFiles/osmpbf-outline.dir/osmpbf-outline.cpp.o] Error code 1

make[2]: stopped in /tmp/OSM-binary
1 error

make[2]: stopped in /tmp/OSM-binary
--- tools/CMakeFiles/osmpbf-outline.dir/all ---
*** [tools/CMakeFiles/osmpbf-outline.dir/all] Error code 2

make[1]: stopped in /tmp/OSM-binary
1 error

make[1]: stopped in /tmp/OSM-binary
*** [all] Error code 2

make: stopped in /tmp/OSM-binary
1 error

make: stopped in /tmp/OSM-binary
```

--- a/tools/osmpbf-outline.cpp
+++ b/tools/osmpbf-outline.cpp
@@ -1,3 +1,6 @@
+// used for 'isatty'
+#include <unistd.h>
+
 // used for va_list in debug-print methods
 #include <stdarg.h>
 

